send-stream.c

```diff
@@ -35,7 +35,7 @@ struct btrfs_send_stream {
 
 	/*
 	 * end of last successful read, equivalent to start of current
-	 * malformated part of block
+	 * malformed part of block
```

Comment:
this might be objectionable...

Reply:
I don't remember exactly, as the stream is a formatted stream of bytes, the 'malformatted' might make sense too, but 'malformed' is not cofusing in this context